src/Tribe/Query.php

```diff
@@ -128,9 +133,9 @@ public static function parse_query( $query ) {
 				: false;
 
 			$query->tribe_is_event_query = ( $query->tribe_is_event
-			                                 || $query->tribe_is_event_category
-			                                 || $query->tribe_is_event_venue
-			                                 || $query->tribe_is_event_organizer )
+				|| $query->tribe_is_event_category
```

Comment:
(1/2) not sure if this was deliberate or automated formatting, but just wanted to flag as a) it's not a functional change and so is in a sense unwanted noise b) neither this new indentation nor the previous indentation really match how we want things to be